src/Pages/AddItem.js

```diff
@@ -1,9 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
-const AddItem = () => (
-  <div>
-    <h1 className="title is-1">Add Item </h1>
-  </div>
-);
+export default function AddItem({ token }) {
+  const [item, setItem] = useState('');
+  const [urgency, setUrgency] = useState(7);
 
-export default AddItem;
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    e.target.reset();
+    db.collection(token).add({
+      item,
+      urgency,
+      lastPurchased: null,
+    });
+    // resets input field after submission
+    setUrgency(7);
+    setItem('');
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="item">
+          <b>Grocery item:</b>
+        </label>{' '}
+        <br />
+        <input
+          type="text"
+          id="item"
+          value={item}
+          required={true}
+          onChange={(e) => setItem(e.target.value)}
+        />{' '}
+        <br />
+        <br />
+        {/* This fieldset is similar to what we wad going on with the div, just better for accessibility */}
```

Comment:
I find comments are best used when explaining someone that might not be obvious. I don't necessarily think using fieldset qualifies for this, since a quick search of the component could tell the dev why it would be used here.

Another thing: I don't really know what div you're talking about here. Perhaps that was another comment that was here before but it was removed? Generally speaking, try to keep comments referring to the code right below it. Otherwise the code around it is almost guaranteed to change but the comment will likely remain unchanged

 We don't have to remove it, just something to keep in mind